src/incubator/WheelPicker/index.tsx

```diff
@@ -280,8 +280,8 @@ const WheelPicker = ({
             snapToInterval={itemHeight}
             decelerationRate={Constants.isAndroid ? 0.98 : 'normal'}
             renderItem={renderItem}
-            getItemLayout={getItemLayout}
-            initialScrollIndex={currentIndex}
+            getItemLayout={Constants.isIOS ? getItemLayout : undefined}
+            initialScrollIndex={Constants.isIOS ? currentIndex : undefined}
```

Comment:
I assume the issue we had was only on iOS? 
What was the issue this code fix?

Reply:
The values return to the start (or some other value) even after a value was selected, it did not occur on my Android (would love a confirmation :) - you can see it in the `DurationPicker` if you remove these props.